### ISSUE TITLE
"Edit this page" links don't support the versioning scheme

### ISSUE THREAD

**Issue by mirons-google** [GoogleCloudPlatform/forseti-security#1801]
https://forsetisecurity.org/docs/latest/setup/install.html has an "edit this page" link of https://github.com/GoogleCloudPlatform/forseti-security/edit/forsetisecurity.org/_docs/latest/setup/install.md, which does not exist.

**Comment by EricBeach** [comment]
The root problem is that in ``_includes/site/layout/docs/tabs.html`` the ``page.path`` variable is used. Publicly, we use ``latest`` while the repo uses ``_latest``

```
<a class="icon github" href="{{ site.github.repository_url }}/edit/forsetisecurity.org/{{ page.path }}">
        Edit this page
      </a>
```

**Comment by mirons-google** [comment]
@EricBeach Yes, -dev should be the branch for changes.

**Comment by EricBeach** [comment]
@mirons-google - SGTM. I assume it should be ``-dev`` and the current code was a mistake. I've fixed it in #1807

**Comment by EricBeach** [comment]
Fixed in #1807
